x-pack/plugins/cross_cluster_replication/public/app/sections/auto_follow_pattern_edit/auto_follow_pattern_edit.js

```diff
@@ -0,0 +1,154 @@
+    apiStatus: PropTypes.string.isRequired,
+  }
+
+  componentWillUnmount() {
+    this.props.clearApiError();
+  }
+
+  renderMissingCluster({ name, remoteCluster }) {
+    const { intl } = this.props;
+
+    const title = intl.formatMessage({
+      id: 'xpack.crossClusterReplication.autoFollowPatternEditForm.emptyRemoteClustersTitle',
+      defaultMessage: 'Remote cluster not found'
+    });
+
+    return (
+      <Fragment>
+        <EuiCallOut
+          title={title}
+          color="warning"
+          iconType="help"
+        >
+          <FormattedMessage
+            id="xpack.crossClusterReplication.autoFollowPatternEditForm.emptyRemoteClustersDescription"
+            defaultMessage="The remote cluster '{remoteCluster}' does not exist or is not connected. Make sure it is connected before editing the '{name}' auto-follow pattern." //eslint-disable-line max-len
+            values={{ remoteCluster, name }}
+          />
+        </EuiCallOut>
+        <EuiSpacer />
+        <EuiButton
+          {...routing.getRouterLinkProps('/list', BASE_PATH_REMOTE_CLUSTERS)}
+          fill
+          iconType="plusInCircle"
+        >
+          <FormattedMessage
+            id="xpack.crossClusterReplication.autoFollowPatternEditForm.viewtRemoteClustersButtonLabel"
+            defaultMessage="View remote clusters"
+          />
+        </EuiButton>
+      </Fragment>
+    );
+  }
+
+  render() {
+    const { createAutoFollowPattern, apiStatus, apiError, autoFollowPattern } = this.props;
+
+    const breadcrumbs = [{
+      text: (
+        <FormattedMessage
+          id="xpack.crossClusterReplication.breadcrumbs.listText"
+          defaultMessage="Cross Cluster Replication"
+        />
+      ),
+      ...routing.getRouterLinkProps('/'),
+    }, {
+      text: (
+        <FormattedMessage
+          id="xpack.crossClusterReplication.autoFollowPattern.breadcrumbs.editText"
+          defaultMessage="Edit auto-follow pattern"
+        />
+      ),
+    }];
+
+    return (
+      <EuiPage>
+        <EuiPageBody>
+          <EuiPageContent >
+            <EuiBreadcrumbs breadcrumbs={breadcrumbs} responsive={false} />
+            <EuiSpacer size="xs" />
+
+            <EuiPageContentHeader>
+              <EuiTitle size="l">
+                <h1>
+                  <FormattedMessage
+                    id="xpack.crossClusterReplication.autoFollowPattern.editTitle"
+                    defaultMessage="Edit auto-follow pattern"
+                  />
+                </h1>
+              </EuiTitle>
+            </EuiPageContentHeader>
+
+            <RemoteClustersProvider>
+              {({ isLoading, error, remoteClusters }) => {
+                if (isLoading) {
+                  return (
+                    <SectionLoading>
+                      <FormattedMessage
+                        id="xpack.crossClusterReplication.autoFollowPatternForm.loadingRemoteClusters"
+                        defaultMessage="Loading remote clusters..."
```

Comment:
I think the copywriters discourage the use of the ellipsis in these kinds of situations: `...`

Reply:
The [eui writing style guide](https://elastic.github.io/eui/#/guidelines/writing) mentions: `Do. Use an ellipsis for truncated text or situations that require waiting.`, talking as an example the loading texts.